src/Buckle/Buckle/CodeAnalysis/Binding/BoundExpressionTypes.cs

```diff
@@ -79,6 +80,8 @@ private BoundBinaryOperator(SyntaxType type, BoundBinaryOperatorType opType, Bou
             BoundTypeClause.Int),
         new BoundBinaryOperator(SyntaxType.GREATER_THAN_GREATER_THAN_TOKEN, BoundBinaryOperatorType.RightShift,
             BoundTypeClause.Int),
+        new BoundBinaryOperator(SyntaxType.GREATER_THAN_GREATER_THAN_GREATER_THAN_TOKEN, BoundBinaryOperatorType.
+            UnsignedRightShift,BoundTypeClause.Int),
```

Comment:
Dont split the line on a accessor operator, so move 'BoundBinaryOperatorType.' to like 84 so line 83 ends in a comma

Reply:
Also make sure there is a space after all commas